### PR TITLE
dev-libs/univalue: Bump to 1.0.3

### PR DESCRIPTION
**Pull request by luke-jr**

### CODE REVIEW

File: dev-libs/univalue/univalue-1.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	./autogen.sh || die
```
Comment (SoapGentoo) [inline dev-libs/univalue/univalue-1.0.3.ebuild]: what does this do? bootstrap autotools?

File: dev-libs/univalue/univalue-1.0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```
Comment (SoapGentoo) [inline dev-libs/univalue/univalue-1.0.3.ebuild]: why no EAPI 6?

File: dev-libs/univalue/univalue-1.0.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	./autogen.sh || die
+}
+
+src_configure() {
+	econf --disable-static
+}
+
+src_install() {
+	default_src_install
+	prune_libtool_files
```
Comment (SoapGentoo) [inline dev-libs/univalue/univalue-1.0.3.ebuild]: use just `default` and if possible eschew `prune_libtool_files`: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives

File: dev-libs/univalue/univalue-1.0.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+SRC_URI="https://codeload.github.com/jgarzik/${PN}/tar.gz/v${PV} -> ${P}.tgz"
+SLOT="0/0"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```
Comment (SoapGentoo) [inline dev-libs/univalue/univalue-1.0.3.ebuild]: the order of the variables should be:
```
DESCRIPTION
HOMEPAGE
SRC_URI
<empty line>
LICENSE
SLOT
KEYWORDS
IUSE
```

File: dev-libs/univalue/univalue-1.0.3.ebuild (hunk 1 above)
Comment (luke-jr) [inline dev-libs/univalue/univalue-1.0.3.ebuild]: Yes

**Comment by luke-jr** [comment]
Addressed comments

File: dev-libs/univalue/univalue-1.0.3.ebuild (hunk 1 above)
Comment (mgorny) [reply dev-libs/univalue/univalue-1.0.3.ebuild]: I suppose what @SoapGentoo was getting at, is that you're supposed to use `eautoreconf` for this.

**Comment by luke-jr** [comment]
Done & rebased

File: dev-libs/univalue/univalue-1.0.3.ebuild [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+SLOT="0/0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf || die
```
Comment (Zlogene) [inline dev-libs/univalue/univalue-1.0.3.ebuild]: there is no need to add die after eautoreconf here

**Comment by mgorny** [comment]
Given how much time this took already, I'm going to fix the one remaining issue and merge it.

**Pull request closed without merging**